lib/membrane/compositor_element.ex

```diff
@@ -43,6 +43,11 @@ defmodule Membrane.VideoCompositor.CompositorElement do
         description:
           "Initial position of the video on the screen, given in the pixels, relative to the upper left corner of the screen",
         default: {0, 0}
+      ],
+      pts_offset: [
+        spec: Membrane.Time.t(),
+        description: "Input stream PTS offset in nanoseconds",
+        default: 0
```

Comment:
I think this should be renamed to `offset` - similarly how this option is named in the `audio_mix_plugin`. This applies to names of variables and fields, too.

Reply:
I've changed it, but I think that `pts_offset` is a more descriptive and less confusing name for it. In `audio_mix_plugin` we don't encounter other types of offsets, but here at first glance, it might be interpreted another way, since the `position` option refers to final video placement.

Reply:
Ok, I agree. `timestamp_offset` is fine, then?

Reply:
Sounds great :)

Reply:
Done